config.yml

```
@@ -0,0 +1,13 @@
+  hosts:
+    - "188.242.205.5"
+  keyspace: "chaika_sales"
+  user: "grisha"
+  password: "n*_irR#2*$h_341nUe"
+
+cassandra-test:
+  hosts:
+    - "188.242.205.5"
+  keyspace: "chaika_sales_test"
+  user: "grisha"
+  password: "n*_irR#2*$h_341nUe"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)